ui/page/transaction/transaction_details_page.go

```diff
@@ -50,6 +50,7 @@ type TxDetailsPage struct {
 	rebroadcastClickable            *decredmaterial.Clickable
 	rebroadcastIcon                 *decredmaterial.Image
 	gtx                             *layout.Context
+	unmined                         bool
```

Comment:
this variable isn't needed, because republishing the transaction doesn't automatically make it mind, it just adds it to a queue of transactions to the network, and it might still get rejected